c7n/resources/securityhub.py

```diff
@@ -83,6 +88,103 @@ def register_resources(klass, registry, resource_class):
         resource_class.filter_registry.register('finding', klass)
 
 
+def get_security_hub(session_factory, retry):
+
+    client = local_session(session_factory).client('securityhub')
+    hub = retry(client.describe_hub, ignore_err_codes=(
+        'InvalidAccessException',))
+    if not hub:
+        return []
+
+    hub.pop('ResponseMetadata')
+    return [hub]
+
+
+@resources.register('security-hub')
+class SecurityHubMember(QueryResourceManager):
+
+    filter_registry = filters
+    action_registry = actions
+
+    class resource_type(TypeInfo):
+        service = 'securityhub'
+        id = name = 'HubArn'
+        arn_type = 'hub'
+        cfn_type = 'AWS::SecurityHub::Hub'
+        enum_spec = ("describe_hub", None, None)
+
+    def resources(self):
+        return self.filter_resources(get_security_hub(self.session_factory, self.retry))
+
+    def get_resources(self, resource_ids):
+        return get_security_hub(self.session_factory, self.retry)
+
+
+@filters.register('valid-master')
```

Comment:
s/valid-master/master, also just make this a value filter derived so it can look at dates/etc

Reply:
thanks makes sense. will change